extensions/amp-instagram/1.0/amp-instagram.js

```diff
@@ -34,36 +32,8 @@ class AmpInstagram extends PreactBaseElement {
     );
     return super.isLayoutSupported(layout);
   }
-
-  /** @override */
-  init() {
-    return dict({
-      'requestResize': (height) => {
-        this.forceChangeHeight(height);
```

Comment:
`forceChangeHeight` is an AMP method, so this should remain here.

Reply:
Done.